### ISSUE TITLE
Negative daemons number.

### ISSUE THREAD

**Issue by marespiaut** [subchannel13/EnchantedFortress#2]
Hello,

I've installed this game via F-Droid and on several occasion the daemon count went negative but the game did not ended.
At some point in the game I inevitably lost, despite having every characteristics to the maximum.

**Comment by subchannel13** [comment]
Found a place where scouts can report negative demon number. It doesn't look possible for the real number of demons to go negative but I've added safeguards anyway.

**Comment by vn971** [comment]
@subchannel13 yes, it's about what scout report says. (Well, one does not normally know know the real number of demons.)

@marespiaut you should research "banish demons" early. I found out that delaying this to later stages makes the final spell be kind of a moving target.
For example, you have constant 40% farming, 0% walls, 30% soldiers, 30% research, but the time to fully research "banish demons" rotates back and fourth around the number of 60 turns.

**Comment by subchannel13** [comment]
Yes, one should not delay banishing demons for too long. If banishment becomes too much of a moving target then try researching something else. Farming will let you live with less farmers and scholarship will flat out speed up all research, including banishment.

**Comment by marespiaut** [comment]
Hello,
I turns out I did not realized you had to dedicate some resources to scholars in order to augment the research level.
I also thought *Banish demons* was optional since the number of turns kept growing due to my low research level and because I was able to eradicate all daemons below zero level.

At some point in the game, the daemons get so strong that you will lost whatever you do. It seems logical as – I suppose – daemons gets very powerful and can overtake humans.

The game is very enjoyable once you figure out the mechanics of it.
I've made a screenshot a few days ago of the daemon number being negative, but I don't know if it's useful or necessary anymore.
